Gruntfile.js

```diff
@@ -90,7 +90,7 @@ module.exports = function(grunt) {
         options: {
           // The src/vendor paths are needed to find the CF components' files.
           // Feel free to add additional paths to the array passed to `concat`.
-          paths: grunt.file.expand('src/vendor/*').concat([])
+          paths: '<%= loc.src %>/vendor/'
```

Comment:
Isn't that what this was already doing, though? Is this just a simpler way of writing it? The expand+concat isn't needed after all?